### PR TITLE
Added example on how to get node's value

### PR DESCRIPTION
**Pull request by kayrus**
/cc @robszumski @yichengq

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -35,12 +35,24 @@ func main() {
...
-	resp, err := kapi.Set(context.Background(), "foo", "bar", nil)
+	// set "/foo" key with "bar" value
+	resp, err := kapi.Set(context.Background(), "/foo", "bar", nil)
 	if err != nil {
 		log.Fatal(err)
 	} else {
+		// print common key info
 		log.Print(resp)
 	}
+	// retrieve "/foo" key's value
```
Comment (yichengq) [inline client/README.md]: s/retrieve/get

File: client/README.md [hunk 2]
```diff
@@ -35,12 +35,24 @@ func main() {
...
+	resp, err = kapi.Get(context.Background(), "/foo", nil)
+	if err != nil {
+		log.Fatal(err)
+	} else {
+		// print common key info
+		log.Print(resp)
+		// print value
+		log.Print(resp.Node.Value)
```
Comment (yichengq) [inline client/README.md]: could this be more descriptive? e.g, key '/foo' value = bar

**Comment by kayrus** [comment]
@yichengq fixed

**Comment by yichengq** [comment]
I tried to run it, and get this:
```
2015/10/13 08:04:46 &{set {Key: /foo, CreatedIndex: 6, ModifiedIndex: 6, TTL: 0} {Key: /foo, CreatedIndex: 4, ModifiedIndex: 4, TTL: 0} 6}
2015/10/13 08:04:46 &{get {Key: /foo, CreatedIndex: 6, ModifiedIndex: 6, TTL: 0} <nil> 6}
2015/10/13 08:04:46 '/foo' key has 'bar' value
```
It is hard to see what is happening from the log line. I hope that it could be more readable.

BTW, you need to form your commit title better. Ref: https://github.com/coreos/etcd/blob/master/CONTRIBUTING.md#format-of-the-commit-message

**Comment by kayrus** [comment]
@yichengq it is much better rather it was before ))) but I agree. what would you like to see?

```
setting key /key with bar value
getting /key value
metadata is ...
/key key has "bar" value
```

**Comment by yichengq** [comment]
I would add quote on key and value string using `%q` in `fmt.Printf`. The sentence looks good to me.

**Comment by kayrus** [comment]
Fixed

File: client/README.md [hunk 3]
```diff
@@ -35,11 +35,25 @@ func main() {
 		log.Fatal(err)
 	}
 	kapi := client.NewKeysAPI(c)
-	resp, err := kapi.Set(context.Background(), "foo", "bar", nil)
+	// set "/foo" key with "bar" value
+	log.Print("Setting '/key' key with 'bar' value")
```
Comment (yichengq) [inline client/README.md]: '/foo'

File: client/README.md [hunk 4]
```diff
@@ -35,11 +35,25 @@ func main() {
...
+	resp, err := kapi.Set(context.Background(), "/foo", "bar", nil)
 	if err != nil {
 		log.Fatal(err)
 	} else {
-		log.Print(resp)
+		// print common key info
+		log.Printf("Metadata is %q\n", resp)
+	}
+	// get "/foo" key's value
+	log.Print("Getting '/key' key value")
```
Comment (yichengq) [inline client/README.md]: Getting the value of '/key' key

File: client/README.md [hunk 5]
```diff
@@ -35,11 +35,25 @@ func main() {
...
-	resp, err := kapi.Set(context.Background(), "foo", "bar", nil)
+	// set "/foo" key with "bar" value
+	log.Print("Setting '/key' key with 'bar' value")
+	resp, err := kapi.Set(context.Background(), "/foo", "bar", nil)
 	if err != nil {
 		log.Fatal(err)
 	} else {
-		log.Print(resp)
+		// print common key info
+		log.Printf("Metadata is %q\n", resp)
```
Comment (yichengq) [inline client/README.md]: Do you think we should mention `Set is done. Metadata is xxxxxxx` here?

**Comment by yichengq** [comment]
don't forget to use our git commit title format: https://github.com/coreos/etcd/blob/master/CONTRIBUTING.md#format-of-the-commit-message

**Comment by kayrus** [comment]
Fixed

**Comment by yichengq** [comment]
LGTM. Thanks for making it easier to understand!

**Pull request merged**